src/Processors/Content/Create.php

```diff
@@ -249,6 +249,12 @@ private function setPageDataMultiple( string $page ) {
 		} else {
 			$this->pageData['overwrite'] = false;
 		}
+
+		if ( isset( $exploded[8] ) && $exploded[8] === 'true' ) {
+			$this->pageData['noseo'] = true;
+		} else {
+			$this->pageData['noseo'] = false;
+		}
```

Comment:
This can be `$this->pageData['noseo'] = isset( $exploded[8] ) && $exploded[8] === 'true';`